SnookR/sublist/templates/sublist/sublist.html

```diff
@@ -4,9 +4,8 @@
 <head>
   <meta charset="utf-8">
 
-  <title>McAnulty and Barry's Sublist</title>
-  <meta name="description" content="McAnulty and Barry's Sublist">
-  <meta name="author" content="Evan Smelser">
+  <title>{{ sublist.name }}'s Sublist</title>
+  <meta name="description" content="{{ sublist.name }}'s Sublist">
```

Comment:
I found these generic html templates online, but was thinking we'd probably wanna change it moving forward. I've heard good things about bootstrap. I'm going to research it and see how it might be able to help moving forward. Do you have any experience verifying or otherwise @bobbyesh ?